src/main/java/uk/gov/pay/ledger/transaction/search/model/RefundSummary.java

```diff
@@ -17,23 +17,26 @@
 
     private Long amountSubmitted;
 
+    private Long amountRefunded;
+
     public RefundSummary() {
     }
 
-    public RefundSummary(String status, Long amountAvailable, Long amountSubmitted) {
+    public RefundSummary(String status, Long amountAvailable, Long amountRefunded) {
         this.status = status;
         this.amountAvailable = amountAvailable;
-        this.amountSubmitted = amountSubmitted;
+        this.amountRefunded = amountRefunded;
+        this.amountSubmitted = amountRefunded;
```

Comment:
what's the plan for `amountSubmitted`? Right now there are two fields in this class that are populated with `amountRefunded`.

Reply:
It should be removed with the old database column, we were somewhat restricted by failing master builds stopping us from updating pact tests